frontend/doc-site/04__Analyze Public Data/4_0__Get Started.mdx

```diff
@@ -1,18 +1,18 @@
 # Analyze Public Data
 
-CellxGene provides several two visual tools to explore the data sets we host. These tools allow for powerful hypothesis exploration with functionalities including subsetting, differential expression, and others to help you explore the data.
+CZ CellxGene provides several visual tools to explore the data sets we host. These tools allow for powerful hypothesis exploration with functionalities including subsetting, differential expression, and others to help you explore the data.
 
 In order to understand which tool will best meet your needs, it’s helpful to understand the difference between our two main tools for exploring hosted data. Once you decide which tool you want to use, click on it’s name to view the tutorials.
 
-## The Hosted Data Explorer
+## CZ CellxGene Discover
 
-The hosted data explorer allows you to _explore individual data sets_ hosted on the CellxGene platform in UMAP form (<NextLink href="/docs/04__Analyze%20Public%20Data/4_1__Hosted%20Tutorials">learn</NextLink>, [example dataset](https://cellxgene.cziscience.com/e/53d208b0-2cfd-4366-9866-c3c6114081bc.cxg/)).
+CellxGene Discover allows you to _explore individual data sets_ hosted on the CellxGene platform in UMAP form (<NextLink href="/docs/04__Analyze%20Public%20Data/4_1__Hosted%20Tutorials">learn</NextLink>, [example dataset](https://cellxgene.cziscience.com/e/53d208b0-2cfd-4366-9866-c3c6114081bc.cxg/)).
 
 <Image src={"/doc-site/4a_tabulasapiens_cldn7.png"} />
 
 ## <NextLink href="/docs/04__Analyze%20Public%20Data/4_2__scExpression%20Documentation/4_2_1__Get%20Started">scExpression</NextLink>
 
 scExpression allows for an _exploration of gene expression by cell type across all
-data sets_ in the CellxGene data corpus. It is currently in beta.
+data sets_ in the CZ CellxGene data corpus. It is currently in beta.
```

Comment:
```suggestion
data sets_ in the CELLxGENE Discover data corpus. It is currently in beta.
```
I like your approach elsewhere to use CZ only once at the top of the document. Replicating that here. The data corpus belongs to Discover, so adding that in.